tests/integration_tests/tests/test_cases.py

```diff
@@ -541,6 +541,22 @@ def _reset_file_system(self):
         self.clear_directory('/opt/manager/resources/blueprints')
         self.clear_directory('/opt/manager/resources/uploaded-blueprints')
 
+    def _get_latest_execution(self, workflow_id):
+        ex_list = self.client.executions.list(
```

Comment:
I'm assuming you mean equivalent rather than aberration?
(for the actual point in question, I'm 50/50- in this context it doesn't feel too unclear to me but I can definitely understand confusion if you're juggling another part of the code at the same time)

Reply:
I meant abbreviation :). I think it's better to always of the clearer side and not conflict with some standards in other parts of the code

Reply:
I keep forgetting to not use "ex". I hope I'll remember it this time :)